pkg/monitortests/kubeapiserver/apiservergracefulrestart/monitortest.go

```diff
@@ -145,7 +161,12 @@ func podFrom(locator string) monitorapi.PodReference {
 }
 
 func (*apiserverGracefulShutdownAnalyzer) EvaluateTestsFromConstructedIntervals(ctx context.Context, finalIntervals monitorapi.Intervals) ([]*junitapi.JUnitTestCase, error) {
-	return nil, nil
+	testName := "[sig-arch] connection(s) to a kube-apiserver instance should not be created very late in the graceful termination process (more than 80% has passed)"
```

Comment:
would it make more sense to make this sig-apimachinery?

In k8s, I find sig-arch to encombassing a wide range of the project and not usually related to a single team. I'd name it to correspond to the team that would triage this if the test failed.

Reply:
yeah, i will remove the just test bits, as @p0lyn0mial pointed out, we already have a test for it and it is marked as `sig-apimachinery`